docs/pages/textarea.js

```diff
@@ -36,219 +53,270 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+          placeholder="Write something about yourself..."
+          label="About me"
+          value={value}
+        />
+      </Box>
+    );
+  }
+      `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Helper text"
+          description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
+        >
+          <MainSection.Card
+            defaultCode={`
+  function Example(props) {
+    const [value, setValue] = React.useState('')
+    return (
+      <Box width="100%">
+        <TextArea
+          id="aboutmemore"
+          onChange={({value}) => setValue(value)}
+          placeholder="Write something about yourself..."
+          helperText="Describe your favorite hobbies, foods, or books."
+          label="About me"
+          value={value}
+        />
+      </Box>
+    );
+  }
+      `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Readonly"
+          description={`
+      A \`TextArea\` can be in read-only mode in order to present information to the user without allowing them to edit the content. Typically they are used to show content or information that the user does not have permission or access to edit.
+    `}
+        >
+          <MainSection.Card
+            defaultCode={`
+  function Example(props) {
+    const [value, setValue] = React.useState('To keep shopping inspirational and actionable, we set high standards for our Merchants. Your website was not approved due to fuzzy, low quality images.');
+    return (
+      <Box width="100%">
+        <TextArea
+          id="aboutmereadonly"
+          onChange={({value}) => setValue(value)}
+          label="Current errors"
+          value={value}
+          readonly
+        />
+      </Box>
+    );
+  }
+      `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Disabled"
+          description={`
+      A \`TextArea\` can be disabled to indicate the user is unable to interact with it, either by mouse or keyboard. Disabled fields do not need to pass contrast requirements, so do not use a \`disabled\` TextArea to present information to the user (use \`readonly\` instead).
+    `}
+        >
+          <MainSection.Card
+            defaultCode={`
+  function Example(props) {
+    const [value, setValue] = React.useState('')
+    return (
+      <Box width="100%">
+        <TextArea
+          disabled
+          id="disabled"
+          onChange={({value}) => setValue(value)}
+          placeholder="Write something about yourself..."
+          label="About me"
+          value={value}
+        />
+      </Box>
+    );
+  }
+      `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Error message"
+          description={`
     A TextArea can display its own error message.
```

Comment:
ditto here and throughout with nits about articles, but this isn't new copy so feel free to ignore